src/modules/Builder/features/SaveAndPublish/SaveAndPublish.hooks.ts

```diff
@@ -202,6 +203,12 @@ export const useSaveAndPublishSetup = (hasPrompt: boolean) => {
       return;
     }
 
+    const isValid = await trigger();
+
+    if (isValid) {
```

Comment:
why did add the password popup on form validity? please check `sendRequestWithPasswordCheck` we will do it there later AND only in case we have no password in secrurity storage. Per my understanding we should add popup with errors in case of form invalidate : https://mindlogger.atlassian.net/wiki/spaces/MINDLOGGER1/pages/169476176/AUS-7.1.0+-+Save+Publish
![image](https://user-images.githubusercontent.com/16320432/230892176-4dd952df-bf70-4f72-9cc4-b3d67b3a0eb9.png)
![image](https://user-images.githubusercontent.com/16320432/230892227-fe87ffc8-6575-41a5-8aa5-4264e078043b.png)
![image](https://user-images.githubusercontent.com/16320432/230892284-eb67eb49-d3d3-4871-a808-4734c0b6310b.png)